qutip/core/semidefinite.py

```diff
@@ -4,6 +4,23 @@
 This module implements internal-use functions for semidefinite programming.
 """
 
+__all__ = [
+    'complex_var',
+    'herm',
+    'pos_noherm',
+    'pos',
+    'dens',
+    'kron',
+    'conj',
+    'bmat',
+    'dag',
+    'memoize',
+    'qudit_swap',
+    'initialize_constraints_on_dnorm_problem',
+    'dnorm_problem',
+    'dnorm_sparse_problem',
```

Comment:
Not all these should be user facing, or even existing...
```suggestion
    'dnorm_problem',
    'dnorm_sparse_problem',
```

Reply:
made the changes, sorry for the error